tests/features/typing_text_test.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from pyupgrade._data import Settings
+from pyupgrade._main import _fix_plugins
+
+
+@pytest.mark.parametrize(
```

Comment:
usually there's a set of noop tests here -- you probably want at least (1) no rewrite for < python 3 (2) unrelated names are not rewritten